src/classes/HealthCheckVMapper.cls

```diff
@@ -73,40 +73,39 @@ public virtual with sharing class HealthCheckVMapper {
         return HealthCheckDefinitionService.getInstance();
     }
 
-    /**********************************************************************************
-    * @description Retrieves the current date associated with a feature model.
-    * @return A string representing the current date.
-    **********************************************************************************/
-    public String updateHealthCheckLastRunDate(){
-        FeatureParameterModel featureParamDateModel = 
-            (FeatureParameterDateModel)this.locateFeatureParameterService().getFeatureParameterModel();
-
-        return featureParamDateModel.convertToString(); 
-    }
-
     /**********************************************************************************
     * @description Retrieve a view model representing a health check to run.
     * @return A HealthCheckVModel representing details of a health check to be run.
     **********************************************************************************/
-    public virtual HealthCheckVModel getHealthCheckViewModel(){
-        FeatureParameterDateModel featureParamDateModel = 
-           (FeatureParameterDateModel)this.locateFeatureParameterService().getFeatureParameterModel();
-        String lastRunDate = featureParamDateModel.convertToString(); 
-        
-        List<HealthCheckDefinitionVModel> healthCheckDefViewModelsList = new List<HealthCheckDefinitionVModel>(); // list to store VIEW MODELS
-        List<HealthCheckDefinitionModel> healthCheckDefinitionList = this.locateHealthCheckDefinitionService().getHealthCheckDefinitionModels(); // retrieve models
+    public virtual HealthCheckVModel getHealthCheckViewModel() {
+        try {
+            FeatureParameterDateModel featureParamDateModel = 
+               (FeatureParameterDateModel)this.locateFeatureParameterService().getFeatureParameterModel('SettingsHealthCheckLastRunDate');
+            String lastRunDate = featureParamDateModel.convertToString();
 
-        for (HealthCheckDefinitionModel healthChkDefModel : healthCheckDefinitionList){
-            //Generate view model for current health check definition model
-            HealthCheckDefinitionVModel healthChkDefViewModel = new HealthCheckDefinitionVModel(healthChkDefModel.name,
-                                                                                                healthChkDefModel.className,
-                                                                                                healthChkDefModel.namespace);
+            if (lastRunDate == '12/31/2015') {
+                lastRunDate = '';
+            }
```

Comment:
Since Date.format() returns an arbitrary format based on the user's locale and other settings, we cannot guarantee the string output. It would suggest setting lastRunDate to an empty string. Then, if featureParamDateModel.value is not that a strongly-typed date of 12/31/2015, set lastRunDate to featureParamDateModel.convertToString().